src/main/java/org/opentripplanner/routing/edgetype/factory/PatternHopFactory.java

```diff
@@ -94,192 +79,60 @@
-                return 0;
-            } else {
-                return 1;
-            }
-        }
-    }
-
-    private double alongTrackDistance(Coordinate coord, double crossTrackError) {
-        double distanceFromStart = SphericalDistanceLibrary.fastDistance(start, coord);
-        double alongTrackDistance = FastMath.acos(FastMath.cos(distanceFromStart / RADIUS)
-            / FastMath.cos(crossTrackError / RADIUS))
-            * RADIUS;
-        return alongTrackDistance;
-    }
-}
-
-class IndexedLineSegmentComparator implements Comparator<IndexedLineSegment> {
-
-    private Coordinate coord;
-
-    public IndexedLineSegmentComparator(Coordinate coord) {
-        this.coord = coord;
-    }
-
-    @Override
-    public int compare(IndexedLineSegment a, IndexedLineSegment b) {
-        return (int) FastMath.signum(a.distance(coord) - b.distance(coord));
-    }
-}
-
 /**
  * Generates a set of edges from GTFS.
  */
 public class PatternHopFactory {
 
     private static final Logger LOG = LoggerFactory.getLogger(PatternHopFactory.class);
 
-    private static final int SECONDS_IN_HOUR = 60 * 60; // rename to seconds in hour
-
-    private static GeometryFactory _geometryFactory = GeometryUtils.getGeometryFactory();
+    private static GeometryFactory geometryFactory = GeometryUtils.getGeometryFactory();
 
     private GtfsFeedId feedId;
 
     private OtpTransitService transitService;
 
-    private CalendarService calendarService;
-    
     private Map<ShapeSegmentKey, LineString> geometriesByShapeSegmentKey = new HashMap<ShapeSegmentKey, LineString>();
 
     private Map<FeedScopedId, LineString> geometriesByShapeId = new HashMap<FeedScopedId, LineString>();
 
-    private Map<FeedScopedId, double[]> distancesByShapeId = new HashMap<FeedScopedId, double[]>();
-    
-    private FareServiceFactory fareServiceFactory;
+    private Map<FeedScopedId, double[]> distancesByShapeId = new HashMap<>();
 
-    private Multimap<StopPattern, TripPattern> tripPatterns = HashMultimap.create();
+    private FareServiceFactory fareServiceFactory;
 
     private GtfsStopContext context = new GtfsStopContext();
 
     // the location types for transfers.txt
-    public static final int STOP_LOCATION_TYPE = 0;
-    public static final int PARENT_STATION_LOCATION_TYPE = 1;
+    private static final int STOP_LOCATION_TYPE = 0;
+    private static final int PARENT_STATION_LOCATION_TYPE = 1;
 
-    public int subwayAccessTime = 0;
+    private int subwayAccessTime = 0;
 
     private double maxStopToShapeSnapDistance = 150;
 
-    public int maxInterlineDistance = 200;
+    private int maxInterlineDistance = 200;
 
     public PatternHopFactory(GtfsContext context) {
         this.feedId = context.getFeedId();
-        this.transitService = context.getOtpTransitService();
-        this.calendarService = context.getCalendarService();
+        this.transitService = context.getTransitBuilder().build();
     }
     
     public PatternHopFactory() {
```

Comment:
This constructor appears to be unused, and I don't see any good reason for it. We should remove it.